R/members.R

```diff
@@ -256,14 +256,18 @@ space_member_remove.data.frame <- function(space, users, remove_projects = NULL,
 #'
 #' Returns the list of members and their usage information for a given space.
 #' You must either be the admin of the space or your role must have permissions
-#' to see the members list.
+#' to see the members list. Result will include `last_activity` only if only
```

Comment:
I find these types of "column will sometimes be present, will sometimes not" can be kinda confusing. I realize you may not have much of a choice here - I just wanted to note that this is tricky to code against. 

Some options/ideas if you want to create consistent function return values:
- create an `_impl` function that is hidden, and expose `less_than_30d`, `more_than_90d` functions with different structures
- Always return a `last_activity` column with consistent types, but `NA` if more than `30d` and a warning

Reply:
Will go with option 2, thanks!